webpack.config.js

```diff
@@ -46,6 +46,10 @@ module.exports = {
           },
         ],
       },
+      {
+        test: /\.plutus$/i,
```

Comment:
I think this should definitely be added to the documentation -- a lot of our users aren't very familiar with webpack

Reply:
Where would this documentation go (what section)?

Reply:
Probably in `doc/plutus-comparison.md`, under 
```
### Working with scripts

#### Using scripts from the frontend
```

Reply:
On second thought, I think that maybe it should be added to `getting-started`